src/device-registry/controllers/events-consumer.js

```diff
@@ -0,0 +1,41 @@
+const { logObject, logElement } = require("../utils/log");
```

Comment:
this looks good, like that we have reduced the layers of folders involved in reaching this controller.

Reply:
👊